src/_assets/css/_bootstrap_config.scss

```diff
@@ -27,7 +27,9 @@ $container-max-widths: (
   xxl: 1330px
 );
 $grid-breakpoints: (
-  xs: 0,
+  0: 0,
+  xxs: 320px,
+  xs: 480px,
```

Comment:
@fertolg - note that I've created two other breakpoints, which I used to style mobile screenshot images. I'm hoping that this doesn't break anything else. If it doesn't I can look at alternate solutions.